charts/kubefed/values.yaml

```diff
@@ -6,19 +6,36 @@
 ##
 controllermanager:
   enabled: true
-  replicaCount: 2
-  repository: quay.io/kubernetes-multicluster
-  image: kubefed
-  tag: canary
-  imagePullPolicy: IfNotPresent
-  logLevel: 2
-  resources:
-    limits:
-      cpu: 500m
-      memory: 512Mi
-    requests:
-      cpu: 100m
-      memory: 64Mi
+  manager:
```

Comment:
suggestion: I would rather name this `controller`.

Reply:
Yes, we could use `controller` too.